localstack-core/localstack/utils/analytics/metadata.py

```diff
@@ -106,7 +106,7 @@ def get_machine_id() -> str:
     return doc["machine_id"]
 
 
-def get_localstack_edition() -> Optional[str]:
```

Comment:
q: Why is this change here necessary? This isn't only a type hint change, but actually modifies what we return (which could have impact on clients using the API).

Reply:
This was another inconsistency discovered when drafting the spec. This function is used by the `/health`, `/info` and `/diagnostics` to return the LS edition. I changed this to consistently return a string value to avoid a special `null` case. True, there is a possibility of this affecting the client in which case it should be reverted.

Reply:
Well, I think since this can't really happen for anyone starting localstack with the official containers, we can keep it as it is now, unifying the return values for all endpoints and simplifying the spec. Thanks for the follow-up, @viren-nadkarni!